CMakeLists.txt

```diff
@@ -148,6 +148,11 @@ if (WITH_FREERDP_DEPRECATED)
 	add_definitions(-DWITH_FREERDP_DEPRECATED)
 endif()
 
+option(WITH_FREERDP_DEPRECATED_COMMANDLINE "Build FreeRDP deprecated command line options" OFF)
```

Comment:
Typo in commit message: `WITH_FREERDP_DEPRECATED_COMMANDLINE_COMMANDLINE` -> `WITH_FREERDP_DEPRECATED_COMMANDLINE`